src/stack_mitosis/interpreter.clj

```diff
@@ -62,6 +78,15 @@
                 [db-id (:TagList (invoke-logged! rds (op/tags arn)))])))
        (into {})))
 
+(defn latest-snapshot
+  "Returns the latest snapshot for an instance"
+  [rds target]
+  (->> (invoke-logged! rds (op/list-snapshots target))
+       (:DBSnapshots)
+       (sort-by :SnapshotCreateTime)
+       (last)
+       (:DBSnapshotIdentifier)))
```

Comment:
No need to adjust, just a comment on clojure style, when using `->>` threading macros, if there are no arguments other than the threaded value, it's ok to leave off the parentheses. Ie:

    (->> snapshots :DBSnapshots (sort-by :SnapshotCreateTime) last :DBSnapshotIdentifier)